terraform/services/backup-plan/terraform.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  backend "s3" {
+    key = "backup-plan/terraform.tfstate"
+  }
+}
+
+provider "aws" {
+  alias  = "primary"
```

Comment:
I'm curious how it works when aliasing multiple providers, but only using one of the aliases. Does terraform just make the first defined provider the default? In any event, dropping the alias here is clearer, unless you wanted to explicitly define every _primary_ resource with the `aws.primary` provider.
```suggestion
```